diagnostic.md

```diff
@@ -7,27 +7,32 @@ Write your answers inside this file, where it's indicated by the comments.
     repository in your answer.
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    git clone git@github.com:byau/git-basics-diagnostic.git
     ```
 
 2.  What do you do after cloning a repository, but before starting work?
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    cd git-basics-diagnostic
     ```
 
 3.  What command do we use to create a new branch? Name this branch `response`
     in your answer.
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    git branch response   // to create a branch
+    git checkout response // move to response branch
+
+    or
+
+    git checkout -b response // create a branch called 'response' and move into it
     ```
 
 4.  We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    git diff
```

Comment:
Although this is correct, we haven't covered `git diff` yet officially; we were looking for `git status`.